cpp/include/cudf/functions.h

```diff
@@ -848,13 +848,13 @@ gdf_error gdf_quantile_aprrox(	gdf_column*  col_in,       //input column with 0
                                 gdf_context* ctxt);        //context info
 
 gdf_error gdf_order_by(gdf_column** input_columns,   //input columns
-                       size_t       num_inputs,      //number of columns in the first parameter (e.g. number of columsn to sort by
+                       size_t       num_inputs,      //number of columns in the first parameter (e.g. number of columsn to sort by)
```

Comment:
Update documentation to fit Doxygen template: https://github.com/rapidsai/cudf/blob/3c7bea328533a78907ab25871d10095e9a838c87/libgdf/src/example_documentation.cpp